e3db/types/note.py

```diff
@@ -65,24 +61,13 @@ def to_json(self):
 
         # Do key names need to match other SDKs to guarantee interoperability?
         to_serialize = {
-            'note_writer_client_id': self.__note_writer_client_id,
-            'mode': self.__mode,
-            'note_recipient_signing_key': self.__note_recipient_signing_key,
-            'note_writer_signing_key': self.__note_writer_signing_key,
-            'note_writer_encryption_key': self.__note_writer_encryption_key,
-            'encrypted_access_key': self.__encrypted_access_key,
-            'type': self.__type,
-            'data': self.data,
-            'plain': self.__plain,
-            'file_meta': self.__file_meta,
+            'note_keys': self.__note_keys.to_json(),
```

Comment:
This look like it will include a private key, which it shouldn't.

Reply:
I don't think it will? `NoteKeys` contains only mode, writer public signing key, writer public encryption key, encrypted access key, and recipient public signing key